extensions/file/src/main/resources/META-INF/mule-module.properties

```diff
@@ -1,4 +0,0 @@
-module.name=file
```

Comment:
Done, there is another branch that has these changes to use runner for file-extension.